ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/JumpPass.scala

```diff
@@ -0,0 +1,22 @@
+package io.joern.ghidra2cpg.passes
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.EdgeTypes
+import io.shiftleft.passes.{CpgPass, DiffGraph}
+import io.shiftleft.semanticcpg.language._
+class JumpPass(cpg: Cpg) extends CpgPass(cpg) {
+
+  override def run(): Iterator[DiffGraph] = {
+    implicit val diffGraph: DiffGraph.Builder = DiffGraph.newBuilder
+    cpg.call
+      .nameExact("<operator>.goto")
+      .where(_.argument.order(1).isLiteral)
+      .foreach { sourceCall =>
+        val destinationAddress = sourceCall.argument.order(1).code.l.headOption.getOrElse("")
```

Comment:
Instead of getting a default address of `""`, this could be written as something like
```
sourceCall.argument.order(1).code.l.headOption match {
  case Some(destinationAddress) =>
    // deal with address properly

  case _ => 
    // Skip this node since we don't know what the address is
}
```
There are different ways to do this stylistically, but an approach that doesn't attempt to continue once we know that the destination address isn't valid seems cleaner.